### PR TITLE
Adds a javascript API to close widget if open.

### PR DESCRIPTION
**Pull request by Shrikant9**
Issue [#426](https://github.com/bigbinary/acehelp/issues/426)

### CODE REVIEW

**Comment by Shrikant9** [comment]
@breezyboa _a

File: app/javascript/Client/Main.elm [hunk 1]
```diff
@@ -116,6 +116,7 @@ type Msg
     | SearchBarMsg SearchBar.Msg
     | ReceivedUserInfo UserInfo
     | OpenArticleWithId ArticleId
+    | MinimizeAppState ()
```
Comment (breezyboa) [inline app/javascript/Client/Main.elm]: No need to create a new Msg for this. Just use `SetAppState Minimized`

File: app/javascript/Client/Main.elm [hunk 2]
```diff
@@ -495,6 +510,7 @@ subscriptions model =
         [ Animation.subscription Animate [ model.containerAnimation ]
         , userInfo <| decodeUserInfo >> ReceivedUserInfo
         , openArticle <| OpenArticleWithId
+        , closeWidget <| MinimizeAppState
```
Comment (breezyboa) [inline app/javascript/Client/Main.elm]:
```
closeWidget <| always (SetAppState Minimized)
```

**Comment by Shrikant9** [comment]
@breezyboa _a
Done.

**Pull request merged**